example_cpp_smart_card_client_app/src/window.js

```diff
@@ -59,7 +60,8 @@ GSC.I18n.adjustAllElementsTranslation();
 // Called when the "close" button is clicked. Closes the window.
 function onCloseWindowClicked(e) {
   e.preventDefault();
-  chrome.app.window.current().close();
+  if (GSC.Packaging.MODE == GSC.Packaging.Mode.APP)
```

Comment:
Perhaps this is not needed, if we don't show the close button in the extension mode at all?